data/ecosystems/s/stacks.toml

```diff
@@ -10,6 +10,7 @@ sub_ecosystems = [
   "hiro-systems",
   "jolocom",
   "lnstxbridge",
+  "Stacks Degens",
```

Comment:
You need to create a sub ecosystem toml file if you want this to pass the build correctly.

Reply:
Was updated in the #381 PR.